src/Infrastructure/Reader/TestTaskYamlReader.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Reader;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\Repository\SettingRepositoryInterface;
+use SprykerSdk\SdkContracts\Entity\SettingInterface;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Yaml\Yaml;
+
+class TestTaskYamlReader extends TaskYamlReader
```

Comment:
We don't a test scope here. Just move logic of paths fetching in separate class `SettingsPathProvider` and `StaticPathProvider` that just passes paths from argument. And separate test scope by a service as you've already done

Reply:
Fixed